src/syft/core/io/virtual.py

```diff
@@ -4,15 +4,17 @@
 (such as one powered by P2P tech, web sockets, or HTTP) should
 execute the exact same functionality but do so over a network"""
 
-from syft.core.common.message import ImmediateSyftMessageWithReply
-from syft.core.common.message import ImmediateSyftMessageWithoutReply
-from syft.core.common.message import EventualSyftMessageWithoutReply
-from ..node.abstract.node import AbstractNode
-from ...decorators import syft_decorator
 from typing import final
 
-from .connection import ServerConnection
-from .connection import ClientConnection
+from syft.core.common.message import (
```

Comment:
Just for everyone's awareness - multi-imports are ok as long as they aren't on the same line. We made this style choice a while ago so that git commit logs are cleaner (so this is ok - just saying)